doc/guides/build/index.rst

```diff
@@ -0,0 +1,132 @@
+.. _build_overview:
+
+Build Overview
+##############
+
+The Zephyr build process can be divided into two main phases: a
+configuration phase (driven by *CMake*) and a build phase (driven by
+*Make* or *Ninja*). This document is describing the build using *Make*
+as example.
+
+Configuration Phase
+*******************
+
+The configuration phase begins when the user invokes *CMake*,
+specifying a source application directory and a board target.
+
+.. figure:: build-config-phase.png
+    :align: center
+    :alt: Zephyr's build configuration phase
+    :figclass: align-center
+    :width: 80%
+
+*CMake* begins by processing the *CMakeLists.txt* file in the application
+directory, which will refer to the *CMakeLists.txt* file in the Zephyr
+top-level directory, which in turn will refer to *CMakeLists.txt* files
+throughout the build tree (directly and indirectly). Its primary
+output is a set of *Makefile*  to drive the build process, but *CMake*
+scripts do some build processing of their own:
+
+**Device tree** Using *cpp*, device-tree specifications (*.dts/.dtsi*
+files) are collected from the target’s architecture, SoC, board, and
+application directories and compiled with *dtc*. Then the build tool
+(scripts/dts) convert this into *.h* files for later consumption.
+
+**Device tree fixup** Files named *dts_fixup.h* from the target’s
+architecture, SoC, board, and application directories are
+concatenated into a single *dts_fixup.h*. Its purpose is to normalize
+constants output in the previous step so they have the names
+expected by the source files in the build phase.
+
+**Kconfig** The build tool reads the *Kconfig* files for the target
+architecture, the target SoC, the target board, the target
+application, as well as *Kconfig* files associated with subsystems
+throughout the source tree. It incorporates the device tree outputs to
+allow configurations to make use of that data. It ensures the
+desired configuration is consistent, outputs *autoconf.h* for the
+build phase.
+
+Build Phase
+***********
+
+The build phase begins when the user invokes make. Its ultimate output
```

Comment:
Since you started using italics for tool names, you should probably continue, so ``*make*``